testing/integration/src/builtin.rs

```diff
@@ -11,20 +12,30 @@ use fvm_shared::ActorID;
 use multihash::Code;
 
 use crate::error::Error::{FailedToLoadManifest, FailedToSetActor, FailedToSetState};
+use crate::{
+    datacap_actor, reward_actor, storagemarket_actor, storagepower_actor, verifiedregistry_actor,
+};
+
+type SingletonCodeCID = (Cid, Cid, Cid, Cid, Cid, Cid, Cid, Cid, Cid, Cid);
```

Comment:
?????????????

Reply:
Ah, why don't we just drop this helper and use the manifest directly? That should simplify things.